automatic/prey/update.ps1

```diff
@@ -26,28 +24,26 @@ function global:au_SearchReplace {
 function global:au_AfterUpdate { Update-Metadata -key 'releaseNotes' -value $Latest.ReleaseNotes }
 
 function global:au_GetLatest {
-  $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
+  $LatestRelease = Get-GitHubRelease -Owner "Prey" -Name "Prey-Node-Client"
```

Comment:
Can we use camelCase for variable names as that's consistent with this script and other code.